includes/theme-compat/header.php

```diff
@@ -1,4 +1,12 @@
 <?php
+// server should keep session data for AT LEAST 1 hour
+ini_set('session.gc_maxlifetime', 3600);
+
+// each client should remember their session id for EXACTLY 1 hour
+session_set_cookie_params(3600);
```

Comment:
I think those should be in the `config.php` file

Reply:
This is currently a test functionality. I've moved it to the mandatory includes file "required-includes.php" that loads on every page. 

In the config file, my idea is to have only and only constants. My idea is to keep it as clean as possible and to have absolutely no functionality in it.